pkgs/data/misc/sing-geoip/default.nix

```
@@ -8,16 +8,21 @@
 let
   generator = buildGoModule rec {
     pname = "sing-geoip";
-    version = "20230512";
+    version = "20240312";
 
     src = fetchFromGitHub {
       owner = "SagerNet";
       repo = pname;
       rev = "refs/tags/${version}";
-      hash = "sha256-Zm+5N/37hoHpH/TLNJrHeaBXI8G1jEpM1jz6Um8edNE=";
+      hash = "sha256-nIrbiECK25GyuPEFqMvPdZUShC2JC1NI60Y10SsoWyY=";
     };
 
-    vendorHash = "sha256-ejXAdsJwXhqet+Ca+pDLWwu0gex79VcIxW6rmhRnbTQ=";
+    vendorHash = "sha256-WH0eMg06qCiVcy4H+vBtYrmLMA2KJRCPGXiEnatW+LU=";
+
+    patchPhase = ''
```

```suggestion
    postPatch = ''
```

Can we do it in a way that doesn't seem so strange? For example, getting a patch from a PR or something like that.

---

The go script is only for building the data in the CI. I patch it to build the data locally. I don't think it makes sense to open a PR to change the purpose of the script.